azure-functions-sdk/src/commands/new_app.rs

```diff
@@ -243,7 +243,7 @@ impl<'a> NewApp<'a> {
             concat!(
                 "azure-functions = \"",
                 env!("CARGO_PKG_VERSION"),
-                "\"\nlog = \"0.4.6\"\n"
```

Comment:
Imposing this dependency seemed a bit heavy handed. I'm happy to put it back if you want